src/main/java/javax/measure/converter/UnitConverter.java

```diff
@@ -71,40 +73,43 @@ protected UnitConverter() {
     public abstract boolean isLinear();
 
     /**
-     * Indicates whether this converter is considered the same as the  
-     * converter specified. To be considered equal this converter 
+     * Indicates whether this converter is considered the same as the
+     * converter specified. To be considered equal this converter
      * concatenated with the one specified must returns the {@link #IDENTITY}.
      *
-     * @param  cvtr the converter with which to compare.
-     * @return <code>true</code> if the specified object is a converter 
+     * @param cvtr
+     *         the converter with which to compare.
+     * @return <code>true</code> if the specified object is a converter
      *         considered equals to this converter;<code>false</code> otherwise.
      */
     public boolean equals(Object cvtr) {
-        if (!(cvtr instanceof UnitConverter)) return false;
-        return this.concatenate(((UnitConverter)cvtr).inverse()) == IDENTITY;        
+        if (this == cvtr) { return true; }
```

Comment:
An unfortunate amount of noise due to formatting. Next time I edit, I'll format all the files as a separate commit. This is the only actual change in this file.